### ISSUE TITLE
nodetool status Load info is incorrect

### ISSUE THREAD

**Issue by asias** [scylladb/scylla#394]
only see load on one server
```
$ time nodetool -p 7199 status
Datacenter: datacenter1
=======================
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address        Load       Tokens  Owns    Host ID                               Rack
UN  172.31.19.251  24825      256     ?       42f39ab0-708c-4782-9a48-9267db779d9d  rack1
UN  172.31.19.250  ?          256     ?       c66606a9-33b3-4dea-a368-e45148836ae8  rack1

Note: Non-system keyspaces don't have the same replication settings, effective ownership information is meaningless

real	0m28.320s
user	0m2.612s
sys	0m0.206s
```

**Comment by tzach** [comment]
@amnonh please take a look. I still see this issue.

**Comment by slivne** [comment]
Fixed by #427
